### PR TITLE
Update cheat sheet

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated cheat sheet link as per [doc](https://docs.google.com/document/d/1uI5hsuhe5VUpFSSiQas6FVB2F7RhlbNXcr5Pvr_Lm2c/edit#heading=h.yaskb2d6l6px)

## QA

- Visit https://multipass-run-323.demos.haus/
- Check that requested change has been made

## Issue / card

Fixes https://warthogs.atlassian.net/browse/WD-9821

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -54,14 +54,12 @@ <h3>VMs for Windows, macOS and Linux</h3>
 <section class="p-strip--suru-accent">
   <div class="row u-equal-height u-vertically-center">
     <div class="col-6 u-align--center u-hide--small">
-      <a aria-controls="modal" href="https://assets.ubuntu.com/v1/bb21c0e8-Ubuntu+Server+CLI+pro+tips+06.01.20.pdf?_ga=2.194029293.1707463146.1680603251-187335696.1680603251" aria-hidden="true" tabindex="-1">
```
Comment (carkod) [inline templates/index.html:57]: I think you are supposed to keep that `_ga...` parameter for tracking. You can confirm with @laszlokajtar

File: templates/index.html [hunk 2]
```diff
@@ -54,14 +54,12 @@ <h3>VMs for Windows, macOS and Linux</h3>
...
-        <img src="https://assets.ubuntu.com/v1/39a8dac8-Ubuntu_Server_CLI_pro_tips_2020-04.jpg" style="height: 280px; width: 400px;" alt="Ubuntu Server CLI pro tips"/>
-      </a>
+      <img src="https://assets.ubuntu.com/v1/39a8dac8-Ubuntu_Server_CLI_pro_tips_2020-04.jpg" style="height: 280px; width: 400px;" alt="Ubuntu Server CLI pro tips"/>
     </div>
     <div class="col-6">
       <h2>Get started with Ubuntu the fast way</h2>
       <p>The “Ubuntu Server CLI cheat sheet” is your fast path to learning the Linux command line - from basic file management to deploying Kubernetes and OpenStack.</p>
-      <a class="p-button--positive" aria-controls="modal" target="blank" href="https://assets.ubuntu.com/v1/bb21c0e8-Ubuntu+Server+CLI+pro+tips+06.01.20.pdf?_ga=2.194029293.1707463146.1680603251-187335696.1680603251">Download Cheat Sheet</a>
```
Comment (carkod) [inline templates/index.html:64]: Or maybe the tracking parameter should be added here if we don't want it wrapping the image.

**Review by carkod — APPROVED** [review]
LGTM, just one comment.

File: templates/index.html (hunk 1 above)
Comment (mtruj013) [reply templates/index.html:57]: Why would we be tracking it here though? Clicking on the image doesn't do anything nor does it route anywhere

File: templates/index.html (hunk 2 above)
Comment (mtruj013) [reply templates/index.html:64]: That would make more sense, I'll leave a comment in the doc as the link I was given did not include ga params

**Pull request merged**